docs/tabular_buckets.md

```diff
@@ -0,0 +1,86 @@
+# Tabular Bucket Types
+## Abstract
+Riak TS is built upon Riak KV which is built upon Riak Core in so much as
+Riak Core was factored out of a monolithic Riak KV and in so much as Riak TS
+utilizes several function call paths of Riak KV but fundamentally differs
+as neither the table nor a record within a table is a Riak Object in the sense
+that consumers as well as developers of Riak KV have committed to memory, the
+rules to reason about to guarantee eventual consistency in an AP system. To
+create a coherent understanding of Riak TS Table and Record entities, we will
+explore the operations that act upon these entities with specific callouts to
+differences between the TS entities and Riak KV ~entities Bucket Type, Bucket,
+Key, and Object.
+
+Subsets of Riak TS operations are exposed via native Erlang, riak-admin, HTTP,
+and protobuf (PB). However, the full set of operations described herein are
+exposed via the SQL subset supported by Riak TS. The SQL language implementation
+is exposed via the HTTP and PB interfaces, so is available to all clients in a
+distributed network setting. For this reason, all client operations herein will
+be described via SQL while all Riak TS internal operations will be described via
+Erlang snippets or message sequence chart.
+
+## Status
+This document only fleshes out table creation, which highlights the role that
+Riak Core plays in storing and disseminating (via plumtree broadcast) the
+table metadata and the role that Riak TS plays in responding to cluster-wide
+broadcast and RPC to compile and surface the compiled state throughout the
+cluster.
+
+## Table Operations
+### Create
+When a client issues a SQL `CREATE TABLE <table_name> ...` statement, Riak TS
+parses and transforms the SQL into a data-definition language (DDL) format of
+the table and requests Riak Core to store the DDL as a new bucket type property.
+
+To ensure the bucket type operation is consistent, the bucket type activation
+request is focused through the Riak Core Claimant. The Claimant orchestrates
+the activity of creating and activating the bucket type which culminates in the
+plumtree broadcast of the bucket type metadata around the cluster.
```

Comment:
Not sure it's accurate to say "culminates". It begins with the broadcast of the metadata, culminates with the toggling of the active flag (which admittedly is another broadcast of metadata, but this still seems a bit misleading).